### ISSUE TITLE
Draft tutorial of reviewing UX

### ISSUE THREAD

**Issue by NewJerseyStyle** [NewJerseyStyle/polis#86]
things should be easy to understand in the UI so user knows what to do next
- [ ] tutorial in reviewing UX against some examples
- [ ] identify common UX mistakes
- [ ] how to report other potential issue to our GitHub Issues/spread sheet
- [ ] add the tutorial into [UX section](https://github.com/CivicTechTO/polis/wiki/Accessibility-Test-Guideline#ux)

**Comment by NewJerseyStyle** [comment]
Some resources found by individual contributor Kory

1.Blind
Screen reader best practice:
https://www.komododigital.co.uk/insights/screen-reader-user-experience-best-practices/
-High contrast

2.Color blindness: 
https://uxdesign.cc/color-blindness-in-user-interfaces-66c27331b858
-Color contrast check on Adobe Photoshop accessibility 
-More contrast on design 

3.Testing voice over on different devices: https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing.html#how-to-test-with-at

4.WCAG checker: https://www.audioeye.com/lp/wcag-web-compliance/?utm_adgroup=WCAG-Accessibility-Check&utm_source=google&utm_medium=paidsearch&utm_campaign=Generic-WCAG-Testing-Free-Site-Scan-97F&utm_term=w3c%20accessibility%20checker&utm_content=scanner-audioeye&hsa_acc=3628656526&hsa_cam=20558467141&hsa_grp=153731295779&hsa_ad=674017683507&hsa_src=g&hsa_tgt=kwd-298247978085&hsa_kw=w3c%20accessibility%20checker&hsa_mt=b&hsa_net=adwords&hsa_ver=3&gad_source=1&gclid=Cj0KCQjwgrO4BhC2ARIsAKQ7zUn4UF2aXRmDQZPsM7ZDcJlhD8Sqxw9pPM3ZRXevoMuBwVirV8_QpeYaArw4EALw_wc

5.A11y on Apple: https://support.apple.com/en-gb/accessibility

6.Testing a11y: https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing.html#how-to-test-with-at

**Comment by NewJerseyStyle** [comment]
Change of plan, we are not going for devtool instruction style. Instead we are going with [Figma Jam and Key criteria](https://github.com/CivicTechTO/polis/wiki/Accessibility-Test-Guideline#ux-accessibility)

I will add the above resources and close this issue
